gqlalchemy/models.py

```diff
@@ -350,16 +350,23 @@ def field_in_superclass(field, constraint):
 
             return None
 
+        def get_base_labels() -> Set[str]:
+            base_labels = set()
+            nonlocal bases
+            for base in bases:
+                if hasattr(base, "labels"):
+                    base_labels = base_labels.union(base.labels)
+            return base_labels
+
         cls = super().__new__(mcs, name, bases, namespace, **kwargs)
+
         cls.label = kwargs.get("label", name)
         if name == "Node":
```

Comment:
```
if name != "Node":
     cls.labels = {cls.label} | get_base_labels() | kwargs.get("labels", set())
```

?

Reply:
And another Question, `|` operator is a shorthand for `union()` method, so why are we using both?

Reply:
Thank you for the suggestion, I replaced the block with one if statement and replaced the `|` operator with the `union()` method.